### ISSUE TITLE
Implement Neural Network from Scratch [Machine Learning - Python]

### ISSUE THREAD

**Issue by HarshCasper** [HarshCasper/NeoAlgo#595]
## 🚀 Feature

- Implement a Toy Neural Network for the MNIST Dataset using only Numpy.
- Train it for some number of epochs and some hyperparameters.
- Plot a live/interactive decision boundary.
- Plot the train and validation metrics such as the loss and the accuracies

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

**Comment by Huzaib** [comment]
Can I take up this issue?

**Comment by rohangawhade** [comment]
I would like to work on this issue.

**Comment by HarshCasper** [comment]
Assigned to you @Huzaib

**Comment by Huzaib** [comment]
How much time do I have to complete it?

**Comment by sahanabalappa** [comment]
I would like to work on this

**Comment by HarshCasper** [comment]
We are logging the scores for the <b>Contributor's Hack 2020</b>. If you have one or more than one PR merged during the program, make sure to fill this [form](https://forms.gle/jG7kDteT5EAz23hk7)
